test/TestUtils.py

```diff
@@ -185,3 +186,32 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+def test_get_yaml_files_no_git_verbose(monkeypatch, capsys):
+    options, _ = cli.get_config(['-v'])
+    monkeypatch.setenv('PATH', '')
+    utils.get_yaml_files(options)
+    stdout = capsys.readouterr().out.split('\n')
+    assert stdout[0] == 'Discovering files to lint: git ls-files *.yaml *.yml'
+    assert stdout[1].startswith('Warning: Failed to locate command:')
```

Comment:
[`str.splitlines()`](https://docs.python.org/3/library/stdtypes.html#str.splitlines) seems like a way to go here. Also, it'd be more readable if you named the lines:
```suggestion
    msg1, msg2 = capsys.readouterr().out.splitlines()
    assert msg1 == 'Discovering files to lint: git ls-files *.yaml *.yml'
    assert msg2.startswith('Warning: Failed to locate command:')
```

Reply:
Exception message might be multiline (end with '\n'). But yep, this is nice, I'll do a variation of that.

Reply:
Maybe match until the exception name then?

Reply:
The full message in my case is `Warning: Failed to locate command: [Errno 2] No such file or directory: 'git': 'git'`.